src/providers/DevicesProvider/AudioInputProvider.tsx

```diff
@@ -74,16 +92,18 @@ const AudioInputProvider: React.FC = ({ children }) => {
           console.log(
             'Previously selected audio input lost. Selecting a default device.'
```

Comment:
At this point you should probably clean this `if` up and unify the error handling:

```
let nextInput: string;
if (…) {
  console.log('… Selecting a default device.');
  nextInput = newAudioInputs[0].deviceId;
} else {
  console.log('… Reselecting input.');
  nextInput = 'default';
}
const nextDevice = await replaceDevice(nextInput);
try {
  await meetingManager.selectAudioInputDevice(nextDevice);
  …
```

Reply:
Good point, will update the code